docs/releases/v2020.1.rst

```diff
@@ -0,0 +1,189 @@
+Gluon 2020.1
+============
+
+This is the first release of Gluon in 2020, based on OpenWrt 19.07. It
+introduces the ath79 target, which will replace ar71xx in the short
+term.
+
+Added Hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+-  devolo WiFi pro 1200e
+-  devolo WiFi pro 1200i
+-  devolo WiFi pro 1750c
+-  devolo WiFi pro 1750e
+-  devolo WiFi pro 1750i
+-  devolo WiFi pro 1750x
+-  GL.iNet GL-AR300M-Lite
+-  OCEDO Raccoon
+-  TP-Link Archer C6 v2
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+-  Aruba AP-303
+-  Aruba Instant On AP11
+-  AVM FRITZ!Repeater 1200
+
+ipq806x-generic
+~~~~~~~~~~~~~~~
+
+-  Netgear R7800
+
+lantiq-xway
+~~~~~~~~~~~
+
+-  AVM FRITZ!Box 7312
+-  AVM FRITZ!Box 7320
+-  AVM FRITZ!Box 7330
+-  AVM FRITZ!Box 7330 SL
+
+lantiq-xrx200
+~~~~~~~~~~~~~
+
+-  AVM FRITZ!Box 7360 (v1, v2)
+-  AVM FRITZ!Box 7360 SL
+-  AVM FRITZ!Box 7362 SL
+-  AVM FRITZ!Box 7412
+
+mpc85xx-p1020
+~~~~~~~~~~~~~
+
+-  Enterasys WS-AP3710i
+-  OCEDO Panda
+
+ramips-mt7620
+~~~~~~~~~~~~~
+
+-  TP-Link Archer C2 (v1)
+-  TP-Link Archer C20 (v1)
+-  TP-Link Archer C20i
+-  TP-Link Archer C50 (v1)
+-  Xiaomi MiWifi Mini
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+-  Netgear EX6150 (v1)
+-  Netgear R6220
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+-  GL.iNet VIXMINI
+-  TP-Link TL-MR3020 (v3)
+-  TP-Link TL-WA801ND (v5)
+-  TP-Link TL-WR902AC (v3)
+
+Removed Hardware support
+------------------------
+
+-  ALFA Network Hornet-UB
+-  ALFA Network Tube2H
+-  ALFA Network N2
+-  ALFA Network N5
```

Comment:
I think we discussed that we want to document the rationale for removed hardware support in the release notes.

Reply:
Added